src/com/example/ppj04/Task01.java

```diff
@@ -0,0 +1,7 @@
+package com.example.ppj04;
+
+public class Task01 {
+    public static void main(String[] args) {
+        System.out.println("Task01");
```

Comment:
pleas move this logic to the separate method

Reply:
fixed